src/server/components/dashboard/controllers.ts

```diff
@@ -327,6 +332,7 @@ export async function listsEditController(
       csrfToken: getCSRFToken(req),
     });
   } catch (error) {
+    logger.error(`listsEditController Error: ${(error as Error).message}`);
```

Comment:
maybe checkout my suggestion on HttpError and exception chaining above

Reply:
@jenbutongit I guess a potential issue with that approach is that I use the technique of logging the function name with the error in 5 different functions of that file, so would I need to create a logger with a new defaultMeta for each function?

I'm new to Winston so maybe there's a better way of doing it. Also... before you answer check out my solution `http500Error()` and let me know if you think it makes things better.

Reply:
Per @jenbutongit 's suggestion, I think you'd use something like below:

```
const logger = createLogger({
  level: LOG_LEVEL,
  format: format.combine(format.errors({ stack: true }), format.json()),
  defaultMeta: { function: "listsEditController" },
  transports: transportsList, // this is defined in services/logger.ts
});


logger.error(error);
```

This is actually a nicer implementation leveraging the out of the box capabilities of Winston resulting in more readable code.  See if this can be done if possible.

Reply:
Hey @alisalaman-cyb, thanks for your input. I still think it would be good to have a chat about this. Looking at the code in your previous answer I see the line:
```
defaultMeta: { function: "listsEditController" },
```

As I said previously, I'm new to Winston but my understanding is that this logger is specific to the `listsEditController` function. If that's the case, would a logger in Winston need to be created for each function that executes a log on error?